src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -678,7 +685,7 @@ internal bool TryGetDataQueryOptions(TexlNode node, bool forCodegen, out DataSou
             if (node.Kind == NodeKind.FirstName
                 && Rule.TexlNodeQueryOptions.Count > 1)
             {
-                if (!(Rule.Document.GlobalScope.GetTabularDataSource(node.AsFirstName().Ident.Name) is IExternalTabularDataSource tabularDs))
+                if (Rule.Document.GlobalScope.GetTabularDataSource(node.AsFirstName().Ident.Name) is not IExternalTabularDataSource tabularDs)
```

Comment:
>not [](http://example.com/codeflow?start=101&length=3)

Are these style preferences or is there a rule now flagging this?

Reply:
This is from [https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0083](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0083)
Of course this is not a strict requirement but it makes the code much easier to read: natural reading + removes the need for extra parenthesis